python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -445,8 +449,8 @@ def runDataRef(self, patchRefList, subdir="", cosmos=None, butlerGen2=None):
         if len(patchRefExistsList) > 0:
             haveForced = True
         else:
-            self.log.warn("No forced dataset exists for, e.g.,: {:} (only showing first dataId in "
-                          "patchRefList).\nPlotting unforced results only.".format(dataId))
+            self.log.warning("No forced dataset exists for, e.g.,: {:} (only showing first dataId in "
```

Comment:
Final comment, I see there are many logging messages that use `.format` -- these all need to be fixed to use deferred % formatting.